hnswlib/hnswalg.h

```diff
@@ -731,13 +717,6 @@ namespace hnswlib {
                 }
             }
 
-            has_deletions_=false;
-
-            for (size_t i = 0; i < cur_element_count; i++) {
-                if(isMarkedDeleted(i))
-                    has_deletions_=true;
```

Comment:
Can we use this to calculate the number of deleted elements instead of storing it?

Reply:
Yes, we can calculate num_deleted_